src/models/user.js

```diff
@@ -20,13 +15,18 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     },
+    username: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true
+    },
     password: {
       type: DataTypes.STRING,
       allowNull: false
     },
     salt: {
       type: DataTypes.STRING,
-      allowNull: false
+      allowNull: true
```

Comment:
The `salt` should not be null.

Reply:
I set salt in true because when i try to create a user with mutation, i get  "message": "notNull Violation: user.salt cannot be null". 
![Screenshot from 2020-01-24 12-13-03](https://user-images.githubusercontent.com/26818116/73079446-fee0f080-3ea2-11ea-8fd1-c84a98a88fe2.png)

And i saw the sample model in block 3, salt has allowNull: true.